README.md

```diff
@@ -235,20 +592,21 @@ namespace Sample
             //Initialize Rollbar
             Rollbar.Init(new RollbarConfig
```

Comment:
Needs to be RollbarLocator.RollbarInstance.Configure(...) instead of Rollbar.Init(...)